src/main/webapp/dev/js/common/const.js

```diff
@@ -83,7 +83,14 @@ const ParamsNames = {
     FEEDBACK_QUESTION_NUMSCALE_STEP: 'numscalestep',
     FEEDBACK_QUESTION_RANKOPTION: 'rankOption',
     FEEDBACK_QUESTION_RANKTORECIPIENTS: 'rankToRecipients',
-
+    FEEDBACK_QUESTION_RANKMINOPTIONSCHECKBOX: 'minOptionsToBeRankedEnabled',
```

Comment:
👍

Reply:
For example, `FEEDBACK_QUESTION_RANK_MIN_OPTIONS_CHECKBOX`. Just separating them by the word will do (We don't need to decide if the naming can be improved).

Reply:
However, currently the style is consistent with those used elsewhere for example :`FEEDBACK_QUESTION_CONSTSUMPOINTSFOREACHRECIPIENT`. And the Java ones (Not added by me) are
```
FEEDBACK_QUESTION_RANKMINRECIPIENTSTOBERANKED
FEEDBACK_QUESTION_RANKMAXRECIPIENTSTOBERANKED
```

Is is necessary to change them, it will cause the style to be inconsistent.

Reply:
Yeah, it will be better to change what you are adding because we don't want to worsen the readability. For the ones you didn't touch it is optional.

The inconsistency doesn't hurt but only further highlight that the ones without the underscores are hard to read.

Reply:
:+1: This can be created as an issue separately then.